tools/proofers/text_frame_std.inc

```diff
@@ -37,7 +37,7 @@ function echo_text_frame_std($ppage)
         "body_attributes" => "id='standard_interface_text' onload='ldAll()'",
     ];
     slim_header(_("Text Frame"), $header_args); ?>
-    <form name="editform" id="editform" method="post" action="processtext.php" target="proofframe" 'onsubmit'="return submitForm()">
+    <form name="editform" id="editform" method="post" action="processtext.php" target="proofframe" onsubmit="return submitForm(this)">
```

Comment:
Does `proof_frame_enh.inc` need this same edit?

Reply:
yes I would imagine the change is very similar.

Reply:
Thanks - I'll do those and push another commit